src/billing/index.js

```diff
@@ -0,0 +1,105 @@
+/* @flow */
+/* @jsx jsxDom */
+/* eslint max-lines: 0 */
+
+import { ZalgoPromise } from 'zalgo-promise/src';
+import { create } from 'xcomponent/src';
+import { type Component } from 'xcomponent/src/component/component';
+
+import { ENV } from '../constants';
+import { getBrowserLocale } from '../lib';
+import { config } from '../config';
+
+import { containerTemplate } from './template';
+
+type BillingOptions = {
+    client : {
+        [string] : (string | ZalgoPromise<string>)
+    },
+    env? : string,
+    locale? : string,
+    logLevel : string,
+    awaitPopupBridge : Function,
+    meta : Object,
+    commit : boolean,
+    token : string
+};
+
+export const BillingPage : Component<BillingOptions> = create({
+    tag:  'billing-page',
+    name: 'billing-page',
+
+    buildUrl(props) : string {
+        let env = props.env || config.env;
+        let { lang, country } = config.locale || getBrowserLocale();
+        const locale = `${ lang }_${ country }`;
+        let isCommit = props.commit ? 1 : 0;
+        const { token } = props;
+
+        return `${ config.inlinedCardFieldUrls[env] }/billing?locale.x=${ locale }&commit=${ isCommit }&token=${ token }`;
```

Comment:
We shouldn't put anything in the url here. This should all be handled using props.

Reply:
Still need to remove query params here.